src/logging/drain.rs

```diff
@@ -0,0 +1,59 @@
+use anyhow::Result;
+use slog::{OwnedKVList, Record};
+
+use super::event;
+
+/// A logging drain designed to integrate with [`slog::Drain`]
+///
+/// The drain can be easily added to a regular [`slog::Logger`]:
+///
+/// ```rust
+/// use kubewarden_policy_sdk::logging;
+/// use slog::{Drain, Logger, o};
+///
+/// let drain = logging::KubewardenDrain::new().fuse();
```

Comment:
Do you think we could already resolve `.fuse`)` on our end? So when the user does e.g. `KubewardenDrain::default()` they can already use the drain, without having to call `fuse()` on it?

Reply:
I would not do `fuse()` on our side, the user might want to chain over drains (to provide filtering capabilities for example)

Reply:
In this case, `::default()` returns a `KubewardenDrain`, since it is a `Default` `impl`, but `::new()` returns a https://docs.rs/slog/2.7.0/slog/struct.Fuse.html. Something like:

```rust
impl Default for KubewardenDrain {
  fn default() -> Self { /* initial values */ }
}

impl KubewardenDrain {
  fn new() -> path::to::slog::Fuse {
    Self::default().fuse()
  }
}
```

Reply:
Good idea, done